langmeta-sqlite/src/main/resources/semanticdb.ddl

```diff
@@ -0,0 +1,49 @@
+create table document(
+  id int,
+  filename text,
+  contents text,
```

Comment:
Yes!! Another thing that I was thinking about when going to sleep ❤️

How about we first try the most straightforward thing - for every start or end, we try to store their: 1) offset, 2) line, 3) column? If this ends up being expensive, we can either remove offset and try again or try line_index instead.

Reply:
How about:
```
Create table documentLine(
  Document int,
  Line int,
  StartChar int,
  EndChar int,
)
```
Then you could just query for a single row using indices instead of a table scan.
```
Select line 
from documentLine dl 
where 
  my_document == dl.document 
  and my_start between dl.startChar and dl.endChar
```
Please ignore all of the casing.  I’m on iOS, it really likes to force capital letters in certain places.